en/api/cciapi.rst

```diff
@@ -891,6 +892,8 @@ cci_connect_with_url
        *   Packet encryption: useSSL = true
        *   Plain text: useSSL = false
 
+    *   **oracleStyleNumberReturn**: If oracleStyleNumberReturn is true, zeros after the decimal point are removed from double and float type results. If it is false, zeros after the decimal point are not removed.
```

Comment:
Please refer to kisoo's comment below and modify it.

Reply:
thanks for the comment I'll fix it.